src/test/java/bio/terra/service/auth/iam/IamServiceTest.java

```diff
@@ -92,6 +92,23 @@ void testDeletePolicyMember() throws InterruptedException {
     assertEquals(policyModel, result);
   }
 
+  @Test
+  void testDeletePolicyMemberAggregateDataReader() throws InterruptedException {
```

Comment:
I think this was a merge error, this is the same as `testDeletePolicyMember()` above. Maybe this method was renamed?